drivers/hopwa_caper/app/models/hopwa_caper/generators/fy2024/generator.rb

```diff
@@ -0,0 +1,162 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HopwaCaper::Generators::Fy2024
+  class Generator < ::HudReports::GeneratorBase
+    def self.fiscal_year
+      'FY 2024'
+    end
+
+    def self.generic_title
+      'HOPWA CAPER'
+    end
+
+    def self.short_name
+      'HOPWA CAPER'
+    end
+
+    def self.file_prefix
+      "v1.0 #{short_name} #{fiscal_year}"
+    end
+
+    def self.default_project_type_codes
+      HudUtility2024.path_project_type_codes
+    end
+
+    def initialize(report)
+      super(report)
+      return unless report&.persisted?
+
+      Rails.logger.level = 1 if Rails.env.development?
```

Comment:
This appears to be equivalent (and I think more clear what's going on)
```suggestion
      Rails.logger.level = :info if Rails.env.development?
```

Reply:
oh yeah, this was just temporary to keep the log chatter down to a minimum, I'll take it out